src/main/java/com/hm/achievement/command/AbstractRankingCommand.java

```diff
@@ -121,6 +115,41 @@ public void executeCommand(CommandSender sender, String[] args) {
 		}
 	}
 
+	private int getPage(String[] args) {
+		int page = 1;
+		if (args.length > 0) {
+			try {
+				page = Integer.parseInt(args[0]);
+			} catch (NumberFormatException ignored) {
+				/* Ignore, use 1 instead. */
+			}
+		}
+		return page;
+	}
+
+	private List<String> getRankingMessages(CommandSender sender) {
+		List<String> rankingMessages = new ArrayList<>();
+		int currentRank = 1;
+		for (Entry<String, Integer> ranking : cachedSortedRankings.entrySet()) {
+			String playerName = Bukkit.getServer().getOfflinePlayer(UUID.fromString(ranking.getKey())).getName();
+			if (playerName != null) {
+				// Color the name of the player if he is in the top list.
+				String color = ChatColor.GRAY + " ";
+				if (sender instanceof Player && playerName.equals(sender.getName())) {
+					color += configColor.toString();
+				}
+				rankingMessages.add(color + getRankingSymbol(currentRank) + " " + playerName + " - " + ranking.getValue());
+			} else {
+				logger.warning("Ranking command: could not find player's name using a database UUID.");
+			}
+			++currentRank;
```

Comment:
Could this be inlined on next line?

Reply:
This is existing code that I refactored into a method, but I can do that.